adcircpy/fort15.py

```diff
@@ -582,15 +623,50 @@ def NSCREEN(self):
             return 100
 
     @property
-    def NWS(self):
+    def NWS(self) -> int:
+        """
+        wind stress number
+        http://adcirc.org/home/documentation/users-manual-v50/input-file-descriptions/nws-values-table/
+        """
+
         if self._runtype == 'coldstart':
-            return 0
-        else:
-            if self.wind_forcing is not None:
+            nws = 0
+        elif self.mesh is not None:
+            wind_forcing = self.mesh._surface_forcing['imetype']
+            if wind_forcing is not None:
                 # check for wave forcing here as well.
-                return self.wind_forcing.NWS
+                nws = int(wind_forcing.NWS % 100)
+        else:
+            nws = 0
+
+        return nws
```

Comment:
NWS parameter extracted from tens place of input configuration coming from the wind forcing class